admin_guide/topics/stateful-sets.adoc

```diff
@@ -0,0 +1,33 @@
+////
+About statefulsets
+
+Module included in the following assemblies:
```

Comment:
Make sure to add the actual assemblies here.

Reply:
@vikram-redhat Absolutely!